plugins/uploadwidget/plugin.js

```diff
@@ -252,6 +252,11 @@
 					oldStyle, newStyle;
 
 				loader.on( 'update', function( evt ) {
+					// Abort if the editor has been destroyed.
+					if ( widget.editor.status === 'destroyed' ) {
+						return;
+					}
+
 					// Abort if widget was removed.
 					if ( !widget.wrapper || !widget.wrapper.getParent() ) {
 						if ( !editor.editable().find( '[data-cke-upload-id="' + id + '"]' ).count() ) {
```

Comment:
In fact it should be enough for fixing this issue to enhance this conditional to check if `editor.editable` exists.